test/routes/address/selectAddressTests.js

```diff
@@ -128,6 +128,7 @@ module.exports = (lab, { permitHolderType, routePath, nextRoutePath, pageHeading
 
           Code.expect(res.statusCode).to.equal(302)
           Code.expect(res.headers['location']).to.equal(nextRoutePath)
+          spy.restore()
```

Comment:
This should be on the sandbox to be consistent with the rest of the spies/stubs.

(Unless there's a particular reason why this is different that I haven't understood?)